go.mod

```diff
@@ -1,9 +1,10 @@
-module github.com/cooktheryan/gitops-primer
+module github.com/kubesaw/export-operator
 
 go 1.17
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20190424111038-f61b66f89f4a
+	github.com/cooktheryan/gitops-primer v0.0.0-20220805191006-8ed0c1266b43
```

Comment:
This doesn't seem to be correct. Since you don't change the imports and you don't use any replaces in go.mod, then it the final code will keep using the code (as a dependency) of the controllers from github.com/cooktheryan/gitops-primer

Reply:
I think the dependency updates will need to be done in multiple steps, first we perform the name change on the main project (i.e. this PR) and after this we can update the references in the go.mod files - there's a few of them not just the one in the root directory.  This is probably required before we can even get the project to build successfully.